snippets/csharp/System.Text.RegularExpressions/Regex/Overview/regex_example1.cs

```diff
@@ -9,7 +9,7 @@
```

Comment:
The comment above would need to be adjusted as well, as it mentions that it is getting the **current** NumberFormatInfo.